### DISCUSSION TITLE
The plugin doesn't compile with 4.26.0

### DISCUSSION THREAD

**Discussion by AmbyNavy** [nxrighthere/UnrealCLR#94] [Help] [answered]
New blank blueprints project: "Project' could not be compiled. Try rebuilding from source manually."

Version 4.26.0.

**Comment by AmbyNavy** [comment, marked as answer]
I fixed this now, just went through the project in Visual Studio and recompiled. It is inconvenient but it works.

**Reply by bobus1432** [reply]
can you show how you did it because i cant figure it out as much as i tried

**Comment by nxrighthere** [comment]
@ClaymoreStudio Can you send me logs of building process before and after success, please?

**Comment by AmbyNavy** [comment]
How do i get those?

**Comment by nxrighthere** [comment]
It should be stored in the `Saved` folder of the project.

**Comment by AmbyNavy** [comment]
[UnrealVersionSelector-2021.01.17-12.47.15.log](https://github.com/nxrighthere/UnrealCLR/files/5826770/UnrealVersionSelector-2021.01.17-12.47.15.log)
[UnrealVersionSelector-2021.01.17-13.19.10.log](https://github.com/nxrighthere/UnrealCLR/files/5826771/UnrealVersionSelector-2021.01.17-13.19.10.log)
[UnrealVersionSelector-2021.01.17-13.21.01.log](https://github.com/nxrighthere/UnrealCLR/files/5826772/UnrealVersionSelector-2021.01.17-13.21.01.log)
[KTAG.log](https://github.com/nxrighthere/UnrealCLR/files/5826773/KTAG.log)
[KTAG-backup-2021.01.17-12.47.02.log](https://github.com/nxrighthere/UnrealCLR/files/5826774/KTAG-backup-2021.01.17-12.47.02.log)
These are all the logs i could find in that folder

**Reply by nxrighthere** [reply]
Sadly, there's no relevant information regarding the failed building process. I see only a successful one.

**Comment by Dejon** [comment]
Just for the record, I have the exact same problem as the OP

**Reply by nxrighthere** [reply]
Without logs of building process, I can't help.

**Reply by Dejon** [reply]
Do you have a pre-compiled plugin folder for 4.26 (and, ideally, 4.27) that you could post a link to? The installation process seems to go OK but when I try to run the UE4 project, I get the dreaded "The following modules are missing or build with a different engine version: UnrealCLR. Would you like to rebuild them now?" I say yes, but after a short while I get the message: "Project could not be compiled. Try rebuilding from source." (same problem as OP)

I have included the log file. It says that the UnrealCLR module is missing and I don't understand why. I've tried to install with the installer and manually on clean new projects in 4.26 and 4.27, and always with the same result. If you had a pre-compiled plugin folder I'd like to try it to see if it solves the problem.

[ATBS_CLR.log](https://github.com/nxrighthere/UnrealCLR/files/7068953/ATBS_CLR.log)

**Reply by nxrighthere** [reply]
I see compilation errors from a 3rd-party module `UE_4.27/Engine/Plugins/Marketplace/UtilityCombatPlugin`, which most likely prevents compilation of the UnrealCLR, so that's why the editor raises an error about the missing module.

**Reply by Dejon** [reply]
That solved it! Thank you so much :)

**Comment by bobus1432** [comment]
Does UnrealCLR work with unreal engine 5.0.3?

**Reply by bobus1432** [reply]
Im asking cuz i get that error and dont know how to fix it.

**Reply by bobus1432** [reply]
this is what i found in the log folder



[MyProject.log](https://github.com/nxrighthere/UnrealCLR/files/9445736/MyProject.log)
[MyProject-backup-2022.08.29-14.32.13.log](https://github.com/nxrighthere/UnrealCLR/files/9445737/MyProject-backup-2022.08.29-14.32.13.log)
[MyProject-backup-2022.08.29-14.54.15.log](https://github.com/nxrighthere/UnrealCLR/files/9445738/MyProject-backup-2022.08.29-14.54.15.log)
[MyProject-backup-2022.08.29-14.55.33.log](https://github.com/nxrighthere/UnrealCLR/files/9445739/MyProject-backup-2022.08.29-14.55.33.log)

**Reply by nxrighthere** [reply]
Please, see https://github.com/nxrighthere/UnrealCLR/pull/248.
